python/pyspark/serializers.py

```diff
@@ -606,7 +604,7 @@ class PickleSerializer(FramedSerializer):
     """
 
     def dumps(self, obj):
-        return pickle.dumps(obj, protocol)
+        return pickle.dumps(obj, pickle.HIGHEST_PROTOCOL)
```

Comment:
Let's poll out this as a constant variable for the possibility that users want to monkeypatch it.

Reply:
Done